Gui/opensim/installer_files/Linux/INSTALL

```diff
@@ -114,27 +119,27 @@ if [[ $OS == "Ubuntu" ]] && [[ $VER == "18.04" ]]; then
 
     [[ $v == 1 ]] && echo "Adding OpenSim to list of desktop applications..."
     echo "[Desktop Entry]
-Version=1.0
-Type=Application
-Name=OpenSim
-Comment=OpenSim is an open source software for neuromusculoskeletal modeling, simulation \
-and analysis.
-Path=$installPrefix
-Exec=$installPrefix/bin/OpenSim
-Icon=opensim
-Terminal=false
-Categories=Science;Education" | sudo tee /usr/share/applications/opensim.desktop > /dev/null
-    sudo cp OpenSimLogoWhiteNoText.png /usr/share/pixmaps/opensim.png
-
-    [[ $v == 1 ]] && echo "Installing opensim-core commands..."
-    cd $installPrefix/bin
-    if [[ $v == 1 ]]; then
-        ./opensim-install-command-line.sh
-    else
-        ./opensim-install-command-line.sh > /dev/null
-    fi
-else
-    err_unsupported_env
+    Version=1.0
+    Type=Application
+    Name=OpenSim
+    Comment=OpenSim is an open source software for neuromusculoskeletal modeling, simulation \
+        and analysis.
```

Comment:
```suggestion
    Comment=Neuromusculoskeletal modeling, simulation, and analysis.
```
We had changed this in the standalone desktop file but I noticed that I had forgot to make that change here as well.